rewrite-java-11/src/main/java/org/openrewrite/java/Java11ParserVisitor.java

```diff
@@ -1358,10 +1358,10 @@ public J visitVariable(VariableTree node, Space fmt) {
         List<JRightPadded<J.VariableDeclarations.NamedVariable>> vars = new ArrayList<>();
 
         for (int i = 0; i < nodes.size(); i++) {
-            VariableTree n = nodes.get(i);
+            JCVariableDecl n = (JCVariableDecl) nodes.get(i);
 
+            boolean implicitlyTyped = skipIfPresent("var");
```

Comment:
See comment on the PR in general for thinking that this could be on `J.VariableDeclarations` instead.

Reply:
I mirrored the structure used in JCTree (They maintain the flag at the variable declaration): 

https://github.com/openjdk/jdk/blob/jdk-11+0/src/jdk.compiler/share/classes/com/sun/tools/javac/tree/JCTree.java#L951-L953

I am not opposed to changing this, maybe we can meet briefly tomorrow so I can better understand your thought process.